exporter/nopexporter/go.mod

```diff
@@ -0,0 +1,78 @@
+module go.opentelemetry.io/collector/exporter/nopexporter
+
+go 1.21
+
+toolchain go1.21.6
+
+require (
+	github.com/stretchr/testify v1.8.4
+	go.opentelemetry.io/collector/component v0.94.1
+	go.opentelemetry.io/collector/consumer v0.94.1
+	go.opentelemetry.io/collector/exporter v0.93.0
+	go.opentelemetry.io/collector/pdata v1.1.0
```

Comment:
@evan-bradley can you please update the dependencies?

Reply:
Thanks for the review, it should be good to go now. The failing check is only related to tests on generated code.